.github/workflows/nvda-test.yml

```diff
@@ -95,12 +95,8 @@ jobs:
           path: "aria-at"
           ref: ${{ inputs.aria_at_ref || 'master' }}
 
-      - name: Checkout aria-at-automation-harness
-        uses: actions/checkout@v4
-        with:
-          repository: "w3c/aria-at-automation-harness"
-          ref: "main"
-          path: "aria-at-automation-harness"
+      - name: Install the ARIA-AT Automation Harness package
+        run: npm install aria-at-automation-harness@"~0.0.1"
```

Comment:
Curious if you think it'd be worth parameterizing the version for the harness

Reply:
I'd rather wait till we need it